amundsenatlastypes/schema/01_column_schema.json

```diff
@@ -40,21 +40,11 @@
   ],
   "entityDefs": [
     {
+      "name": "Column",
       "superTypes": [
-        "Referenceable"
+        "DataSet"
```

Comment:
why did you change the supertype of Column ? Referencable seemed like appropriate choice

Reply:
it used to be DataSet. The confusion is introduced by the diff itself:
```
{
  "entityDefs": [
    {
      "name": "Column",
      "superTypes": [
        "DataSet"
      ]
    }
  ]
}
```